templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Car</h1>
+
+  <p>Are you sure you want to delete the car: {{ car }}?</p>
+  <p><i>All Drivers with this car will be also deleted!</i></p>
+
+  <form action="" method="post">
+  {% csrf_token  %}
+  <input class="btn btn-danger" type="submit" value="Yes, delete">

```

Comment:
it will be a good option for the user to cancel deletion , that’s why we have an extra step before deleting something